example-dotnet/Program.cs

```diff
@@ -11,8 +22,107 @@ static void Main(string[] args)
             BCrypt.Net.BCrypt.Verify("wrong-password", hashedPassword);
+
+            // The next API calls to four libraries are meant to
+            // demonstrate update advisor and auto pull request
+            // features.
+            //
+            // The first two are the cases where the update versions
+            // are of the same frameworks as the currently-used
+            // version.
+            // +--------------------+--------------+----------------+-----------+-----------------+----------+
+            // | Dependency         | Current      | Current        | Update    | Update          | Breaking |
+            // |                    | Version      | Framework(s)   | Version   | Framework(s)    | Update   |
+            // +====================+==============+================+===========+=================+==========+
+            // | recurly-client-api | 1.8.0        | net35          | 1.8.1 or  | net35           | No       |
+            // |                    |              |                | 1.11.4    |                 |          |
+            // +--------------------+--------------+----------------+-----------+-----------------+----------+
+            // | DotNetZip          | 1.10.0       | net20          | 1.11.0    | net20           | Maybe    |
+            // +--------------------+--------------+----------------+-----------+-----------------+----------+
+            //
+            // The other two are the cases where the update versions
+            // have different frameworks than the current versions.
+            // +--------------------+--------------+----------------+-----------+-----------------+----------+
+            // | Dependency         | Current      | Current        | Update    | Update          | Breaking |
+            // |                    | Version      | Framework(s)   | Version   | Framework(s)    | Update   |
+            // +====================+==============+================+===========+=================+==========+
+            // | SharpZipLib        | 1.0.0-alpha1 | netstandard1.3 | 1.0.0-rc1 | net45,          | Maybe    |
+            // |                    |              |                |           | netstandard2.0  |          |
+            // +--------------------+--------------+----------------+-----------+-----------------+----------+
+            // | SharpCompress      | 0.17.1       | net35, net45,  | 0.21.0    | net35, net45,   | Yes      |
+            // |                    |              | netstandard1.0 |           | netstandard1.0, |          |
+            // |                    |              |                |           | netstandard2.0  |          |
+            // +--------------------+--------------+----------------+-----------+-----------------+----------+
+
+            // Usage of recurly-client-api API
+
+            // This does not a break update from 1.8.0 to either 1.8.1 or

```

Comment:
"... is not a breaking update ..."?

```suggestion
            // This is not a breaking update from 1.8.0 to either 1.8.1 or
```

Reply:
@dariusf This if fixed. Thanks!